src/pages/Graph/SummaryPanel.tsx

```diff
@@ -109,7 +109,8 @@ class SummaryPanel extends React.Component<MainSummaryPanelPropType, SummaryPane
               rateInterval={this.props.rateInterval}
             />
           ) : null}
-          {this.props.data.summaryType === 'group' ? (
+          {this.props.data.summaryType === 'box' ? (
+            // TODO, how to handle non-app box?
```

Comment:
Hmm... oh!
You should create a GitHub issue for this... or at least, list it as a bullet in the Epic just to not forget it.

Reply:
Added subtask to epic https://github.com/kiali/kiali/issues/1833